base/memory/shared_memory_tracker.h

```diff
@@ -22,6 +22,9 @@ class ProcessMemoryDump;
 
```

Comment:
Lets add below, after header file inclusions, (after L.13)

// TODO temporarily disable for resolving resource loading issue with networkservice
#define DISABLE_MULTI_CONNECTION_CHANGES 0